tests/qa-integration/pmm_pgsm_integration_test.js

```diff
@@ -154,6 +155,27 @@ Scenario(
   },
 );
 
+Scenario(
+  'PMM-T2261 - Verify Postgresql Dashboard Instance Summary has Data with socket based service and Agent log @not-ui-pipeline @pgsm-pmm-integration',
+  async ({
+    I, dashboardPage, adminPage,
+  }) => {
+    I.amOnPage(dashboardPage.postgresqlInstanceSummaryDashboard.url);
+    dashboardPage.waitForDashboardOpened();
+    await dashboardPage.applyFilter('Service Name', pgsm_service_name_socket);
+    await dashboardPage.expandEachDashboardRow();
+    I.click(adminPage.fields.metricTitle);
+    adminPage.performPageDown(5);
+    adminPage.performPageUp(5);
+    dashboardPage.verifyMetricsExistence(dashboardPage.postgresqlInstanceSummaryDashboard.metrics);
+    await dashboardPage.verifyThereAreNoGraphsWithNA();
+    await dashboardPage.verifyThereAreNoGraphsWithoutData(1);
+    let log = await I.verifyCommand(`docker exec ${container_name} cat pmm-agent.log`);
+    assert.ok(!log.includes('Error opening connection to database \(postgres'),
```

Comment:
Could you please use https://www.npmjs.com/package/codeceptjs-chai assert here? `I.assertFalse(...)`

Reply:
Thanks , addressed in latest commit, pls have a look.